homeassistant/components/goodwe/const.py

```diff
@@ -0,0 +1,26 @@
+"""Constants for the Goodwe component."""
+from datetime import timedelta
+
+from homeassistant.components.number import DOMAIN as NUMBER_DOMAIN
+from homeassistant.components.select import DOMAIN as SELECT_DOMAIN
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+
+DOMAIN = "goodwe"
+
+PLATFORMS = [NUMBER_DOMAIN, SELECT_DOMAIN, SENSOR_DOMAIN]
```

Comment:
Please use the `Platform` enum instead to populate this list.

https://github.com/home-assistant/core/blob/b483754ad3713d1033c88d7f5031850b65d310b7/homeassistant/const.py#L22

Reply:
Done